docs/configuring-metabase/caching.md

```diff
@@ -7,121 +7,107 @@ redirect_from:
 
-If you're self-hosting Metabase, cached question results will be saved to your [application database](../installation-and-operation/configuring-application-database.md).
+For example, if your data only updates once a day, there's no point in querying the database more than once a day, as they data won't have changed. Returning cached results can be significantly faster, as the database won't have to recompute the results to load your question.
 
-If you're using Metabase Cloud, cached question results will be saved to Metabase's servers in the United States.
+## Cache invalidation policies
 
-## Last updated at
+These policies determines how long cached results will be stored.
 
-Questions that use the cache will display a "last cached at" timestamp in the question's **info** panel.
+- [Duration](#duration-caching-policy)
+- [Schedule](#schedule-caching-policy)
+- [Adaptive](#adaptive-caching-policy)
+- [Don't cache results](#dont-cache-results)
 
-## Getting fresh results
+### Duration caching policy
 
-To override a cached question result, re-run the question using the **refresh** button (counterclockwise arrow).
+Keep the cache for a number of hours. When someone runs a query, Metabase will first check if it's cached the results. If not, it runs the query and caches the results for as long as you set the duration.
 
-## Average query execution time
+### Schedule caching policy
 
-Your Metabase instance keeps track of how long it takes each question to run. The average query execution time is used in your [caching settings](#caching-settings).
+Pick when to regularly invalidate the cache. Metabase will periodically run the query according to the schedule you set and store the results. Other people running queries have no effect on when the results are refreshed; they'll always get the cached results unless they explicitly [clear the cache](#clearing-the-cache).
 
-On [paid plans](https://www.metabase.com/pricing/), you can view statistics about query execution time from your [auditing tools](../usage-and-performance-tools/audit.md).
+You can schedule caching to invalidate:
 
-## Caching settings
+- Hourly
+- Daily
+- Weekly
+- Monthly
 
-You can tell Metabase when and what to cache from **Admin settings** > **Caching**:
+We do not yet support lunar cycles.
 
-- [Minimum query duration](#minimum-query-duration)
-- [Cache time-to-live (TTL) multiplier](#cache-time-to-live-ttl-multiplier)
-- [Max cache entry size](#max-cache-entry-size)
+### Adaptive caching policy
 
-### Minimum query duration
+Use a query’s average execution time to determine how long to cache the query's results.
 
-Metabase uses this number to decide whether a question will be cached or not.
+- **Minimum query duration**: Metabase will cache this question if it has an average query execution time greater than this many seconds.
+- **Multiplier**: Metabase will cache questions with an average query execution time greater than this many seconds.
```

Comment:
I think an example would help here. Like question takes in avg 10s to run. On a 100 multiplier, we will store the cache for 1000 seconds or 16 minutes